reports/Washington.md

```diff
@@ -175,6 +175,7 @@ id: wa-seattle-14
 
 * [Video showing individual getting hit with projectile (0:11), falling down, and becoming unresponsive. Medics carry them away from police.](https://twitter.com/chaseburnsy/status/1269890344331571201)
 * [Firsthand account from the medic](https://www.reddit.com/r/Seattle/comments/gywxhz/folks_i_need_your_help/)
+* https://imgur.com/a/fWkU0SA
```

Comment:
Looks right, but how did you confirm it was the 7th?

Reply:
Based on the twitter video and medic first-hand account.